resources/views/user/personal-information.blade.php

```diff
@@ -0,0 +1,110 @@
+{{-- Personal information --}}
+@can('viewPersonalInformation', $user)
+<div class="card">
+    <div class="card-content">
+        <div class="card-title">@lang('user.personal_information')</div>
```

Comment:
This appears in English in Hungarian locale.

Reply:
Yes, I noticed, the whole user locale contains english translations. This should be fixed, but I left it for some other issue, will open one.